shoop/testing/factories.py

```diff
@@ -206,9 +206,11 @@ def post(self, create, extracted, **kwargs):
             sp.categories = shop.categories.all()
 
 
-def get_address(**overrides):
+def get_address(immutable=False, **overrides):
     data = dict(DEFAULT_ADDRESS_DATA, **overrides)
-    return Address(**data)
+    if immutable:
+        return ImmutableAddress.objects.create(**data)
+    return MutableAddress.objects.create(**data)
```

Comment:
I think that cant be Address.from_data but yes it could use MutableAddress.from_data and ImmutableAddress.from_data.